__tests__/app.test.js

```diff
@@ -131,4 +131,67 @@ describe("CORE: GET /api/articles error test suite", () => {
             expect(body.msg).toBe("Path not found.");
         })
     });
+});
+describe("CORE: POST /api/articles/:article_id/comments functionality test suite", () => {
+    test("returns a 201 POST with a comment object of the posted comment to the given article", () => {
+        return request(app)
+        .post("/api/articles/1/comments")
+        .send({ "username": "lurker", "body": "BANANA!" })
+        .expect(201)
+        .then(({ body }) => {
+            expect(body.comment).toHaveProperty("author");
+            expect(body.comment).toHaveProperty("body");
+            expect(body.comment).toHaveProperty("article_id");
+            expect(body.comment).toHaveProperty("comment_id");
+            expect(body.comment).toHaveProperty("created_at");
+            expect(body.comment).toHaveProperty("votes");
+        })
+    })
+});
+describe("CORE: POST /api/articles/:article_id/comments error test suite", () => {
+    test("returns a 400 with an error message when passed a valid article id but user does not exist", () => {
+        return request(app)
+        .post("/api/articles/1/comments")
+        .send({ "username": "lurker55555", "body": "BANANA!" })
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Bad request.");
+        })
+    })
+    test("returns a 400 with an error message when passed a valid article id that does not exist", () => {
+        return request(app)
+        .post("/api/articles/9999/comments")
+        .send({ "username": "lurker", "body": "BANANA!" })
+        .expect(400)
```

Comment:
this should really be a 404 - id is not found

Reply:
On ticket 6 Mezz said I wouldn't need anything from ticket 6 to complete ticket 7 when I raised the issue with merge conflicts. Does this mean I can do this without using the checkArticleExists function I have in ticket 6 (now merged into main). I had to start ticket 7 before merging ticket 6.